lib/iris/tests/unit/coords/test_CellMeasure.py

```diff
@@ -0,0 +1,54 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""Unit tests for the :class:`iris.coords.CellMeaasure` class."""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+import numpy as np
+
+from iris.coords import CellMeasure
+from iris.tests import mock
+from iris.coord_systems import GeogCS
+
+
+class Test_CellMeasure(tests.IrisTest):
```

Comment:
This test class is fairly sparse. At the least you should also have a test for setting the measure to something other than `area` or `volume`.

Given that you're only (currently) testing settable Cell Measures properties, you could replace `self.measure` below with a mocked cell measure object and confirm that the mocked object exhibits the appropriate behaviour without having to explicitly define a measure instance.